<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NinePatchFrame</class>
 <widget class="QFrame" name="NinePatchFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>九宫图</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelEdge">
       <property name="font">
        <font>
         <family>等线 Light</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>边长</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxEdge</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxEdge">
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCommit">
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetResult">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="iconSize">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>10</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>10</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>10</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>10</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxEdge</tabstop>
  <tabstop>pushButtonCommit</tabstop>
  <tabstop>tableWidgetResult</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
